pennylane_lightning/lightning_qubit/lightning_qubit.toml

```diff
@@ -116,7 +117,7 @@ runtime_code_generation = false
 # If the device supports adjoint
 quantum_adjoint = true
 # If the device supports quantum control instructions natively
-quantum_control = false
+quantum_control = true
```

Comment:
I'm curious how this makes sense to be true?

Reply:
There was a small discussion here: https://github.com/PennyLaneAI/pennylane-lightning/pull/615#discussion_r1492826930

The meaning `quantum_control` was roughly determined to be "supported gates can be receive an optional vector of qubits which correspond to control qubits". There was also a discussion on slack saying that the above meaning was now the case? Happy to learn if that is not the case.

Reply:
I think the meaning is arbitrary controls (wires and values) are supported for all gates, which Lightning does with this PR.

Reply:
It would be great to have a more meaningful name for this variable, and also some update to the docstring above.

Reply:
I think changing the name is something we can do, but that is out of scope for this PR. That would affect other toml files stored in Catalyst. Thanks for the suggestion at @AmintorDusko! We will be having some changes in toml files in the coming weeks, specifically regarding `quantum_control` so we will keep it in mind.

Reply:
Thanks all. In the interest of keeping things maintainable, let's avoiding keeping docs on Slack and elsewhere and make sure they are documented herein where possible. For me, I'd have no idea how this is consumed based on the name alone, and the associated value. I'll mark as resolved, but we can plan to update docs in future.